apps/server/src/modules/user-login-migration/service/user-login-migration.service.ts

```diff
@@ -114,6 +115,8 @@ export class UserLoginMigrationService {
 
 		if (!sanisSystem) {
 			throw new InternalServerErrorException('Cannot find SANIS system');
+		} else if (school.systems?.includes(sanisSystem.id as string)) {
```

Comment:
Please make this logic for all migrations and look if the targetSystemId ist already included in the systems array.
Dont use the sanis system